SAMtags.tex

```diff
@@ -80,27 +81,30 @@ \section{Standard tags}
   {\tt MC} & Z & CIGAR string for mate/next segment\\
   {\tt MD} & Z & String for mismatching positions \\
   {\tt MF} & ? & Reserved for backwards compatibility reasons \\
+  {\tt MI} & Z & Molecular-barcode (corrected) sequence or identifier. \\
```

Comment:
I think this needs to change to something like `An identifier which uniquely represents the source molecule within the BAM file without reference to other fields or tags`.  The way we (and I think 10X) are using this in practice is by taking the UMIs and position info, to determine molecular identity, then stamping each molecule's reads with a file-wide unique id.  That could be `chr:pos:pos:BX`, but it generally cannot be just the corrected sequence.  We (fgbio) use a serial number.